snippets/ztp_ReduceOLMFootprint.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collect-profiles-config
+  namespace: openshift-operator-lifecycle-manager
+data:
+  pprof-config.yaml: |
+    disabled: True
+    
```

Comment:
extra space??

Reply:
I think it's generally best practice to leave a space at the end of a file. Snippets perhaps are a bit different but doesn't really make much difference in the output